lib/content/gittenify.js

```diff
@@ -0,0 +1,36 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
```

Comment:
We could use a "real" ES6 [`Map`](https://babeljs.io/docs/learn-es2015/#map-set-weak-map-weak-set) here, just for ES6's sake. But I don't know, how that would play with Chrome's storage. Maybe like this: http://www.2ality.com/2015/08/es6-map-json.html